index.html

```diff
@@ -160,11 +244,13 @@ <h5 class="connect">LET'S CONNECT</h5>
             </ul>
           </section>
           <div><button type="button" class="my-resume-button">Get my resume</button></div>
```

Comment:
- [ ] Please create a style for hover and active for this button. It's a project requirement to use the Figma template styling.